### ISSUE TITLE
Blood flow rate => lung has generic parmaeter with Tags?

### ISSUE THREAD

**Issue by msevestre** [Open-Systems-Pharmacology/PK-Sim#1346]
@Yuri05 Do you see a problem if I define a tag that would be implemented by all organs whose blood flow flows directly into lung?
That way we can use a sum formula, similar to body weight and have new organs (such as Thyroid) incorporated automatically

**Comment by msevestre** [comment]
@Yuri05 Goingto implement it
The TAG will be SPLANCHNIC

**Comment by msevestre** [comment]
https://www.sciencedirect.com/topics/medicine-and-dentistry/splanchnic-blood-flow

**Comment by Yuri05** [comment]
@msevestre 
I would suggest NOT to introduce a new tag. Lung blood flow can be realized with existing tags e.g. like this:
![grafik](https://user-images.githubusercontent.com/25061876/68951495-7d902200-07be-11ea-8720-07decf0f7766.png)
That's how it would look like in the simulation with the new definition
![grafik](https://user-images.githubusercontent.com/25061876/68951577-ab756680-07be-11ea-8a57-291535a5ea70.png)

**Comment by Yuri05** [comment]
now every new organ having the tag "TissueOrgan" and a blood flow rate parameter would be automatically considered in the lung blood flow

**Comment by msevestre** [comment]
works for me!

**Comment by msevestre** [comment]
I don't get it? Do you want me to have to add the tag TissueOrgan to all Blood flow rate parameter? I don't understand this criteria
Should not that be in container TissueOrgan?

**Comment by msevestre** [comment]
oh yeah I forgot that a parameter also has the parent tags now :)

**Comment by msevestre** [comment]
Reopening issue because the condition is not accurate.
I am really in favor of adding a new tag that describes what the formula does as opposed to messing around with NOT and Include etc.. 
@Yuri05 Unless there is a good reason not to add a tag that is physiologically accurate, (SPLANCHNIC) I am going to go ahead and do it

**Comment by PavelBal** [comment]
I do not think `SPLANCHNIC` is the correct tag.

**Comment by StephanSchaller** [comment]
> Reopening issue because the condition is not accurate.
Why is it not correct to use the tag conditions @Yuri05 suggested? 

> I do not think SPLANCHNIC is the correct tag.
I agree. Splanchnic only applies to gut, liver and spleen

**Comment by msevestre** [comment]
yes of course. SPLANCHNIC was just to account for the Q_pve part which is also a sum formula in disguise.

At any rate, I have solved it in a different way.
In general, having tag such as "TissueOrgan" on a parameter is not a good idea. It was required before because we did not have "InContainer" or "NotInContainer" conditions. My goal for new Formula is to be explicit about what they do instead of implicit and ultimately hard to read and udnderstand. That's why, if we can use physiological terms to describes set of organs that belongs together, this will be a win in my opinion

**Comment by StephanSchaller** [comment]
> My goal for new Formula is to be explicit about what they do instead of implicit and ultimately hard to read and udnderstand.

Yes, this would be nice. However, you also have to consider, that if this becomes more, you might have to add a lot of tags to new organs? This might bring its own disadvantages.

**Comment by msevestre** [comment]
I don't see how. We have 20 organs and 2000 parameters. Now we duplicate all tags on most parameters so that we can do stupid stuff like setting "TissueOrgan" on a blood flow parameter. 

But if you have an example of what disadvantage it could bring, please explain

**Comment by StephanSchaller** [comment]
> Now we duplicate all tags on most parameters so that we can do stupid stuff like setting "TissueOrgan" on a blood flow parameter

What's the difference with "Splanchnic" that you added?... now you duplicate this as well?

**Comment by StephanSchaller** [comment]
Isn't "TissueOrgan" a tag you need anyway?

> But if you have an example of what disadvantage it could bring, please explain

General rule ;-) not to make things more complex than required… but maybe I am misunderstanding what you did here ;-)

**Comment by msevestre** [comment]
I would like to have tag defined where they belong. 
Having a parameter with the tag "TissueOrgan" is simply wrong because a parameter is not a TissueOrgan. This is something that we did because we had no other choices but in the future, especially if we redesign the DB for OSMOSES, we should make sure to clean that up as well

Again, as I said previously, I solved the problem in a different way. I did not add any new tag. But for the sake of simplicity, that we all thrive to achieve, we should define tags at the right level in the future. I believe that this would make stuff simpler to understand
